frontend/src/lib/constants/environment.constants.ts

```
@@ -12,21 +12,20 @@ export const SNS_AGGREGATOR_CANISTER_URL: string | undefined =
     ? undefined
     : (import.meta.env.VITE_AGGREGATOR_CANISTER_URL as string);
 
-interface FEATURE_FLAGS {
+export interface FEATURE_FLAGS {
   ENABLE_SNS_2: boolean;
   ENABLE_SNS_VOTING: boolean;
   ENABLE_SNS_AGGREGATOR: boolean;
   ENABLE_CKBTC_LEDGER: boolean;
   ENABLE_CKBTC_RECEIVE: boolean;
 }
 
-export const {
-  ENABLE_SNS_2,
-  ENABLE_SNS_VOTING,
-  ENABLE_SNS_AGGREGATOR,
-  ENABLE_CKBTC_LEDGER,
-  ENABLE_CKBTC_RECEIVE,
-}: FEATURE_FLAGS = JSON.parse(
+/**
+ * DO NOT USE DIRECTLY
+ *
+ * @see feature-flags.store.ts to use feature flags
+ */
+export const FEATURE_FLAGS: FEATURE_FLAGS = JSON.parse(
```

What do you think about having json.parse in the try catch block and set all the flags to false. To not break the whole app in case of wrong flags configuration?

---

That's a good idea. We can do that in another PR

---

I disagree. We should fail fast rather than silently doing something unexpected and confusing.
It's also one of our principles: